diff --git a/node_modules/ajv-formats/dist/formats.js b/node_modules/ajv-formats/dist/formats.js
index e345121..403e5e7 100644
--- a/node_modules/ajv-formats/dist/formats.js
+++ b/node_modules/ajv-formats/dist/formats.js
@@ -170,4 +170,3 @@ function regex(str) {
         return false;
     }
 }
-//# sourceMappingURL=formats.js.map
\ No newline at end of file
diff --git a/node_modules/ajv-formats/dist/index.js b/node_modules/ajv-formats/dist/index.js
index 5db64fe..846705d 100644
--- a/node_modules/ajv-formats/dist/index.js
+++ b/node_modules/ajv-formats/dist/index.js
@@ -34,4 +34,3 @@ function addFormats(ajv, list, fs, exportName) {
 module.exports = exports = formatsPlugin;
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.default = formatsPlugin;
-//# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/ajv-formats/dist/limit.js b/node_modules/ajv-formats/dist/limit.js
index 7f6c7fd..39d1046 100644
--- a/node_modules/ajv-formats/dist/limit.js
+++ b/node_modules/ajv-formats/dist/limit.js
@@ -66,4 +66,3 @@ const formatLimitPlugin = (ajv) => {
     return ajv;
 };
 exports.default = formatLimitPlugin;
-//# sourceMappingURL=limit.js.map
\ No newline at end of file
